packages/dom-ready/src/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Specify a function to execute when the DOM is fully loaded.
+ *
+ * @param {Function} callback A function to execute after the DOM is ready.
+ *
+ * @returns {void}
+ */
+const domReady = function( callback ) {
+  if ( document.readyState === 'complete' || ( document.readyState !== 'loading' && !document.documentElement.doScroll ) ) {
```

Comment:
At least that's what I take from https://developer.mozilla.org/nl/docs/Web/API/Document/readyState